Find a functionally equivalent code:android.content.RestrictionEntry.getTitle:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the user-visible title for the entry, if any.
 * @return the user-visible title for the entry, null if none was set earlier.
 */

@@ -1,3 +1,3 @@
 {
-    return title;
+    return mTitle;
 }

